cmd/main.go

```diff
@@ -19,10 +19,10 @@ package main
 import (
 	"flag"
 	"fmt"
+	configs "github.com/openebs/zfs-localpv/pkg/config"
```

Comment:
nit: Any reason for changing it to configs? The package name is config.
Also please run `goimports` to sort the imports.

Reply:
I have run imports and now it is fixed @Abhinandan-Purkait

Reply:
I have changed to configs because I want to use the QuotaType variable in the config package and it has a conflict with config variable name @Abhinandan-Purkait